process/meetings.md

```diff
@@ -0,0 +1,26 @@
+# Meetings
+
+Meeting can be the most effective way how reach agreement. But too many of meetings can killing the company.
```

Comment:
n_: `But too many meetings can kill the company`

Reply:
:face_with_head_bandage: I've changed that piece multiple time and see that ended up with a complete garbage. Thank you!